### PR TITLE
wr-335: Fix loader display issue on the Feed/My Article page

### PR DESCRIPTION
**Pull request by PablloZz**
image

### CODE REVIEW

File: frontend/src/pages/articles/components/articles-list/articles-list.tsx [hunk 1]
```diff
@@ -19,24 +20,32 @@ const ArticlesList: React.FC<Properties> = ({
...
+
+  return (
+    <Loader isLoading={isLoading && !articles.length} type="circular">
+      <InfiniteScroll
+        hasMore={hasMore}
+        className={styles.articles}
+        dataLength={articlesLength}
+        isLoading={isLoading}
+        onFetchData={onFetchData}
+      >
```
Comment (andanf-e) [inline frontend/src/pages/articles/components/articles-list/articles-list.tsx:36]: Inside InfiniteScroll - there is a Loader passed through - isn't it suitable?

Comment (andanf-e) [reply frontend/src/pages/articles/components/articles-list/articles-list.tsx:36]: I am about loader wrapping Infinite Scroll

Comment (PablloZz) [reply frontend/src/pages/articles/components/articles-list/articles-list.tsx:36]: At first I've tried without the loader, but for some reason the loader inside the infinite scroll doesn't display on the first fetch

Comment (andanf-e) [reply frontend/src/pages/articles/components/articles-list/articles-list.tsx:36]: Okay, thanks

**Review by andanf-e — APPROVED** [review]

**Pull request merged**